src/DonationSummary/resources/js/summary.js

```diff
@@ -1,22 +1,40 @@
-import accounting from 'accounting';
-
 /**
  * @since 2.17.0
  */
 window.GiveDonationSummary = {
     init: function () {
         GiveDonationSummary.initAmount();
         GiveDonationSummary.initFrequency();
-        GiveDonationSummary.initFees();
         GiveDonationSummary.initTotal();
     },
 
+    /**
+     * This function returns formated donation amount.
+     *
+     * @unreleased
+     * @return {string}
+     */
+    getFormattedDonationAmount: function($form) {
+        const unFormattedAmount = Give.fn.unFormatCurrency(
+            $form.find('[name="give-amount"]').val(),
+            Give.form.fn.getInfo('decimal_separator', $form)
+        );
+
+        return Give.fn.formatCurrency(
+            unFormattedAmount,
+            {symbol: Give.form.fn.getInfo('currency_symbol', $form)},
+            $form
+        );
+    },
+
     /**
      * @since 2.17.0
      */
     initAmount: function () {
+        const getFormattedDonationAmount = this.getFormattedDonationAmount;
+
         GiveDonationSummary.observe('[name="give-amount"]', function (targetNode, $form) {
-            $form.find('[data-tag="amount"]').html(GiveDonationSummary.format_amount(targetNode.value, $form));
+            $form.find('[data-tag="amount"]').html(getFormattedDonationAmount($form));
```

Comment:
Let's remove this local instance `const getFormattedDonationAmount = this.getFormattedDonationAmount;` in favor of the parent method.

```suggestion
            $form.find('[data-tag="amount"]').html(GiveDonationSummary.getFormattedDonationAmount($form));
```